test/suites/dev/moonbase/test-staking/test-staking-locks12.ts

```diff
@@ -8,12 +8,11 @@ import {
   alith,
   generateKeyringPair,
 } from "@moonwall/util";
-import { fromBytes } from "viem";
-import { chunk } from "../../../../helpers";
+import { chunk, getDelegatorStakingFreeze } from "../../../../helpers";
 
 describeSuite({
-  id: "D023477",
-  title: "Staking - Locks - bottom and top delegations",
+  id: "D013475",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.